Misc fixes

sameersbn opened the pull request.

Commit:
specify namespace in `kubectl get svc` commands in NOTES.txt

Commit:
mediawiki: fix default value of `mediawikiName` in README.md

lachie83 reviewed (commented):
LGTM

lachie83 commented:
@k8s-bot ok to test

prydonius commented on stable/uchiwa/templates/NOTES.txt:
```diff
@@ -9,7 +9,7 @@ Getting Started:
 {{- else if contains "LoadBalancer" .Values.serviceType }}
 **** NOTE: It may take a few minutes for the LoadBalancer IP to be available.                      ****
-****       You can watch the status of by running 'kubectl get svc -w {{ template "fullname" . }}' ****
+****       You can watch the status of by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "fullname" . }}' ****
```
should we remove the `****` in this one?

sameersbn replied:
I don't think it's necessary. I would leave it as-is.

lachie83 replied:
The stars do cause formatting issues when rendered. Would you mind removing them from the NOTES.txt in this PR?

sameersbn commented:
One thing I noticed is that most of our pvc resource templates do not define any label metadata. Any particular reason for it?

lachie83 commented:
> One thing I noticed is that most of our pvc resource templates do not define any label metadata. Any particular reason for it?

Good catch. We should clean that up and add the standard helm metadata

Commit:
apply standard metadata labels to PVC's

Commit:
removed `*` from `uchiwa/templates/NOTES.txt`

sameersbn commented:
@lachie83 @prydonius updated the PR to:
- rebased PR
- remove the `*` characters from the NOTES.txt files (tried to format the output a little)
- added standard metadata labels to the PVC resources

prydonius commented:
@k8s-bot ok to test

prydonius commented on stable/drupal/templates/NOTES.txt:
```diff
@@ -1,6 +1,5 @@
-*** PLEASE BE PATIENT: Drupal may take a few minutes to install ***
-*******************************************************************
+
+PLEASE BE PATIENT: Drupal may take a few minutes to install
```
We should keep the asterisks in this one. It's only when we're using this with templating that it gets unaligned and looks weird.

sameersbn commented:
done!

On Saturday 19 November 2016 05:13 AM, Adnan Abdulhussein wrote:

> _@prydonius_ commented on this pull request.
> 
> ---
> 
> In stable/drupal/templates/NOTES.txt
> https://github.com/kubernetes/charts/pull/209#pullrequestreview-9328090:
> 
> > @@ -1,6 +1,5 @@
> > -*******************************************************************
> > -**\* PLEASE BE PATIENT: Drupal may take a few minutes to install ***
> > -*******************************************************************
> > +
> > +PLEASE BE PATIENT: Drupal may take a few minutes to install
> 
> We should keep the asterisks in this one. It's only when we're using
> this with templating that it gets unaligned and looks weird.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/kubernetes/charts/pull/209#pullrequestreview-9328090,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AAZCI4F5Q-XdDrUS7_ZjTJAvD_Omjnu8ks5q_jgEgaJpZM4KqgBC.

## 

Sameer Naik

w: http://www.damagehead.com e: sameer@damagehead.com

lachie83 commented:
@k8s-bot ok to test

prydonius commented on stable/jenkins/templates/NOTES.txt:
```diff
@@ -9,7 +9,7 @@
 {{- else if contains "LoadBalancer" .Values.Master.ServiceType }}
 **** NOTE: It may take a few minutes for the LoadBalancer IP to be available.                      ****
-****       You can watch the status of by running 'kubectl get svc -w {{ template "fullname" . }}' ****
+****       You can watch the status of by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "fullname" . }}' ****
```
Sorry if I wasn't clear, the asterisks should be removed from this block because of the command's varying length

sameersbn replied:
didn't notice these were present in the `jenkins chart too. fixed it there now. please take a look.

prydonius commented:
Tests are failing due to the failed deletion count issue, but otherwise the charts are installed without error.

Commit:
removed `*` from `jenkins/templates/NOTES.txt`

sameersbn commented:
rebased and resolved conflicts

viglesiasce commented:
Testing now...

@sameersbn in the future can you send smaller PRs (one per chart), it will make testing and merging much quicker and easier.

viglesiasce commented:
Noticed that there are no version bumps here. At least the charts with the PVC changes need one.

viglesiasce commented:
Manually tested install and basic app verification.

sameersbn commented:
> in the future can you send smaller PRs

Sorry about that. I'll update PR with version updates. Thanks for your review.

Commit:
bump chart version with PVC metadata label additions

sameersbn commented:
latest commit bumps the version for charts with PVC metadata label updates

lachie83 commented:
LGTM

lachie83 commented:
@k8s-bot ok to test

sameersbn commented:
Rebased PR

sameersbn commented:
Is there anything I can do to expedite the merge of the changes in this PR?
Do you want me to split the PR in smaller ones?

viglesiasce commented:
@sameersbn looks like there are merge conflicts again.

sameersbn commented:
rebased to resolve the conflicts

prydonius commented:
@viglesiasce any other changes needed here?

viglesiasce commented:
Not on my end.

On Tue, Dec 6, 2016, 8:37 PM Adnan Abdulhussein <notifications@github.com>
wrote:

> @viglesiasce <https://github.com/viglesiasce> any other changes needed
> here?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/charts/pull/209#issuecomment-265338672>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAZCp3DxEudoG49t4RxQDxZgaPvRLlZ1ks5rFhvXgaJpZM4KqgBC>
> .
>

prydonius commented:
Thanks @sameersbn!
